experimental/PropertyDDS/packages/property-properties/src/propertyFactory.js

```diff
@@ -1103,22 +1162,25 @@ PropertyFactory.prototype._computeTypeid = function (in_propertiesEntry, in_pare
  * @param {string=}                     [in_propertiesEntry.context]    - Context in which the property is created
  * @param {Object=}                     [in_propertiesEntry.properties] - Context in which the property is created
  * @param {number}                      [in_propertiesEntry.length]     - The length of an array property
- * @param {property-properties.BaseProperty=}  in_parent                      - The parent property which will be used as
+ * @param {property-properties.BaseProperty}  in_parent                      - The parent property which will be used as
  *                                                                        the root to construct the property template
- * @param {string|undefined} in_scope - The scope in which the property typeid is defined
- * @param {property-properties.BaseProperty.PathFilteringOptions} [in_filteringOptions]
- *    The options to selectively create only a subset of a property. Creates all properties if undefined.
+ * @param {string} [in_scope] - The scope in which the property typeid is defined
+ * @param {boolean} [in_evaluateConstants] - If constants need to be traversed and created
  *
  * @return {property-properties.BaseProperty|undefined} the property instance
  */
 PropertyFactory.prototype._createFromPropertyDeclaration = function (
```

Comment:
Missing parameter

Reply:
Interfaces changed. We never used filtering option anyway